### DISCUSSION TITLE
App::max_size_points limits webGL canvas to 1024px width silently -- Make settable?

### DISCUSSION THREAD

**Discussion by AriYoung00** [emilk/egui#453] [Ideas]
Hello all --
Just spend a couple hours banging my head against the wall trying to figure out why I couldn't make the webGL canvas for my app grow wider than 1024 pixels. Turns out, `App::max_size_points` returns `[1024, 2048]` by default, limiting the maximum possible width to 1024 pixels. Note that this is called on every (I think?) render cycle, so changing the canvas size manually with `egui_web::resize_canvas_to_screen_size` is not possible either. I understand that the width is limited in order to preserve performance in some browsers; however, I believe that `max_size_points` should probably be settable by the user, rather than an arbitrary hard-coded limit. My application, for example, does not work properly at this width. If this is something that would be well-received, I would be more than happy to put together a PR making the requisite changes to the `App` trait.
Thanks!

**Comment by emilk** [comment]
Just override `max_size_points` in your `impl` for `App` and pick whatever max-size you want, just like the demo app does: https://github.com/emilk/egui/blob/master/egui_demo_lib/src/wrap_app.rs#L53-L55

This is a common question, so what is needed is some better communication of this, the question is where to put that. Where did you go looking for a solution? 

EDIT: I have improved the docs in https://github.com/emilk/egui/commit/ec6268e4ac2f95410cddd661894b7e3b5df987a3

**Reply by AriYoung00** [reply]
Whoops, my bad, major brain fart there. I think the first place that I looked was the documentation for `App::max_size_points`, although I also came across `egui_web::resize_canvas_to_screen_size`. Generally, I think it could be beneficial to add more information on controlling window/canvas size, as I found that both of those I had to mostly figure out from method docs.
